roles/build_openstack_packages/templates/projects.ini.j2

```diff
@@ -29,16 +29,19 @@ use_components={{ cifmw_bop_use_components }}
 # We are only setting these for downstream case when cifmw_bop_osp_release
 # is defined. For downstream case, we are hardcoding release_numbering=minor.date.hash
 # and then just set the release_minor value based on the OSP version
-# (e.g. 0 for 16/16.0, 1 for 16.1, 2 for 16.2 etc).
+# (e.g. 0 for 18/18.0, 1 for 18.1, 2 for 18.2 etc).
 release_numbering=minor.date.hash
 release_minor={{ '0' if '.' not in cifmw_bop_osp_release.split('-')[1] else cifmw_bop_osp_release.split('-')[1].split('.')[1] }}
+nonfallback_branches=^master$,^rpm-master$,^rhos-
+custom_preprocess={{ cifmw_bop_build_repo_dir }}/DLRN/scripts/set_nvr.sh,{{ cifmw_bop_build_repo_dir }}/DLRN/scripts/patch_rebaser.sh
 
 [downstream_driver]
 info_files=osp.yml
 versions_url=https://trunk.rdoproject.org/centos9-{{ cifmw_bop_openstack_release }}/current/versions.csv
-downstream_distro_branch={{ cifmw_bop_osp_release }}{{ '.0' if '.' not in cifmw_bop_osp_release else '' }}-rhel-8-trunk
+downstream_source_git_branch={{ cifmw_bop_osp_release }}-trunk-patches
+downstream_distro_branch={{ cifmw_bop_osp_release }}{{ '.0' if '.' not in cifmw_bop_osp_release else '' }}-rhel-9-trunk
 downstream_tag=osp-{{ cifmw_bop_osp_release.split('-')[1] }}{{ '.0' if '.' not in cifmw_bop_osp_release else '' }}
-downstream_distgit_key=osp-distgit
```

Comment:
This should be:
downstream_distgit_key=osp-distgit (wrong)
downstream_source_git_branch=rhos-18.0-trunk-patches
downstream_source_git_key=osp-patches (it's missing)

downstream_distro_branch=rhos-18.0-rhel-9-trunk (it's fine)
downstream_tag=osp-18.0 (it's fine)
downstream_distgit_key=osp-distgit (it's wrong)
downstream_source_git_key=osp-patches (it's missing)
downstream_source_git_branch=rhos-18.0-trunk-patches (it's fine)